handlers/Blog/components/ArticlesList/index.js

```diff
@@ -2,41 +2,46 @@
 
 require('./styles.css');
 
-import React from 'react';
-import marked from 'marked';
+import React, {PropTypes, Component} from 'react';
+import markdown from 'markdown';
 import {Resolver} from 'react-resolver';
 import {Link} from 'react-router';
 import qs from 'querystring';
+import {scrollTo} from 'FilterBar';
 import Typography from 'Typography';
 import api from 'api';
 import * as fmt from 'fmt';
 
-const DEFAULT_PARAMS = {
+const DEFAULT_QUERY = {
   page: '1',
 };
 
-class ArticlesList extends React.Component {
-  render(): ?ReactElement {
-    var params = Object.assign(
-      {}, DEFAULT_PARAMS,
-      this.context.router.getCurrentParams()
+class ArticlesList extends Component {
```

Comment:
article urls now use query params for paging. Would like to change this back to `/blog/page/:page` and `/blog/tags/:tag/page/:page` style in an upcoming iteration. Just needed to get it done.